src/main/java/bookstore/repository/BookRepository.java

```diff
@@ -0,0 +1,11 @@
+package main.java.bookstore.repository;
+
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.awt.print.Book;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
create required methods

```suggestion
@Repository
public interface BookRepository {
```